components/sst_importer/src/sst_importer.rs

```diff
@@ -995,8 +1018,10 @@ impl SstImporter {
         }
 
         // perform iteration and key rewrite.
-        let mut key = keys::data_key(new_prefix);
-        let new_prefix_data_key_len = key.len();
+        let mut data_key = keys::DATA_PREFIX_KEY.to_vec();
+        let data_key_prefix_len = data_key.len();
```

Comment:
Is it easier to read here if it is written as `let data_key_prefix_len = keys::DATA_PREFIX_KEY.len() ` ?